contrib/ohi-release-notes/action.yml

```diff
@@ -40,41 +49,58 @@ runs:
         git-root: ${{ inputs.git-root }}
         markdown: ${{ inputs.git-root }}/CHANGELOG.md
         yaml: ${{ inputs.git-root }}/changelog.yaml
+    - name: Check if the release is empty
+      id: empty
+      uses: newrelic/release-toolkit/is-empty@v1
+      with:
+        yaml: ${{ inputs.git-root }}/changelog.yaml
+    - name: Abort releasing if the changelog is empty
+      if: ${{ steps.empty.outputs.is-empty == 'true' && inputs.fail-if-empty == 'true' }}
+      shell: bash
+      run: |
+        echo "Aborting release, changelog is empty is-empty=${{ steps.empty.outputs.is-empty }}"
+        exit 0
```

Comment:
I considered them as separate steps, which could fail separately, does it make sense to you?

Reply:
Even if they are different steps (and can fail separately), if we performed both checks before failing we would have both outputs populated.

Reply:
🤔 Maybe the `exit 0` + the "Aborting release..." are confusing us (because they don't abort the job). If that's so, I gues the order doesn't matter.

Reply:
I added the exit 0 as it was defined in the task itself, but as I commented [above](https://github.com/newrelic/release-toolkit/pull/147#discussion_r1137279288) I remove it since it might lead to confusion and allowed it to run both actions independently.